contracts/transmuter/src/contract.rs

```diff
@@ -811,6 +823,16 @@ impl Transmuter {
         })
     }
 
+    #[sv::msg(query)]
+    pub fn get_incentive_pool_balances(
+        &self,
+        ctx::QueryCtx { deps, .. }: ctx::QueryCtx,
+    ) -> Result<GetIncentivePoolBalancesResponse, ContractError> {
+        let balances = self.incentive_pool.get_all_pool_balances(deps.storage)?;
```

Comment:
Sanity checking: no risk of gas issues here, becuase this is bounded by number of tokens in alloy